<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #F5E3CB;
}
</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>581</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget { background-color: transparent; }
QHeaderView::section { background-color: transparent; }
QTableCornerButton::section { background-color: transparent; }

QTableWidget { border-color: rgba(0, 0, 0, 36); }
QTableWidget { gridline-color: rgba(0, 0, 0, 36); }
font: 350 9pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_car">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_customer">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_assign">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>581</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonCars">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>245</red>
            <green>227</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>52</red>
            <green>75</green>
            <blue>97</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
      </property>
      <property name="text">
       <string>Cars</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCustomer">
      <property name="styleSheet">
       <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
      </property>
      <property name="text">
       <string>Customers</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAssign">
      <property name="styleSheet">
       <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
      </property>
      <property name="text">
       <string>Assign</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>581</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButtonImport">
      <property name="styleSheet">
       <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
      </property>
      <property name="text">
       <string>Import Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonExport">
      <property name="styleSheet">
       <string notr="true">QPushButton {

font: 300 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
color: #344B61;
    border: 2px solid #F57B56; 
    border-radius: 5px;

}

QPushButton:hover {
    background-color: #344B61; 
    color: #ffffff; 
}


</string>
      </property>
      <property name="text">
       <string>Export Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelJson">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 125);</string>
   </property>
   <property name="text">
    <string>.JSON</string>
   </property>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>image_car</zorder>
  <zorder>image_customer</zorder>
  <zorder>image_assign</zorder>
  <zorder>labelJson</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
